tests/psp22wrapper.tests.ts

```diff
@@ -0,0 +1,145 @@
+import { expect, setupContract, fromSigner } from './helpers'
+
+describe('MY_PSP22_WRAPPER', () => {
+    async function setup() {
+        let psp22 = await setupContract('my_psp22', 'new', '1000', 'TOKEN', 'TKN', 2)
+        let wrapper = await setupContract('my_psp22_wrapper', 'new', psp22.contract.address)
+        return { psp22, wrapper }
+    }
+
+    it('Deposit for works', async () => {
+        const { psp22, wrapper } = await setup()
+        const { contract: wrapperContract, query: wrapperQuery } = wrapper
+        const { contract: psp22Contract, query: psp22Query, defaultSigner: sender } = psp22
```

Comment:
a quick note: you can just export the psp22.contract as psp22 and wrapper.contract as wrapper, and use like
```typescript
const { psp22, wrapper } = await setup()

psp22.query.balanceOf(...)...
wrapper.tx.depositFor(...)...
```

Reply:
yeah idk I think this way is more transparent